### PR TITLE
Do not create duplicate MEV reward events

### PR DESCRIPTION
**Pull request by LefterisJP**
If the relay reports fee recipient and mev fee recipient and amounts as the same then this means no different MEV reward actually happened so do not create one.

Also go through already existing events in the upgrade and remove duplicates.

### CODE REVIEW

File: rotkehlchen/tests/db/test_db_upgrades.py [hunk 1]
```diff
@@ -1575,6 +1595,10 @@ def test_upgrade_db_37_to_38(user_data_dir):  # pylint: disable=unused-argument
     assert cursor.execute('SELECT * from evm_internal_transactions ORDER BY value DESC').fetchall() == expected_internal_txs  # noqa: E501
     assert cursor.execute('SELECT COUNT(*) FROM used_query_ranges WHERE name=?', (aave_range_key,)).fetchone()[0] == 0  # noqa: E501
     assert cursor.execute('SELECT COUNT(*) FROM sqlite_master WHERE type="table" AND name="aave_events";').fetchone()[0] == 0  # noqa: E501
+    # Make sure that duplicate events were removed
+    expected_history_events = [expected_history_events[0]] + expected_history_events[2:6]
+    assert cursor.execute('SELECT * from history_events WHERE entry_type=4;').fetchall() == expected_history_events  # noqa: E501
+    expected_eth_staking_events_info = [expected_eth_staking_events_info[0]] + expected_eth_staking_events_info[2:6]  # noqa: E501
```
Comment (yabirgb) [inline rotkehlchen/tests/db/test_db_upgrades.py:1601]: is soemthing missing here? I only see an assignation and the test ends

File: rotkehlchen/chain/ethereum/modules/nft/nfts.py [hunk 2]
```diff
@@ -171,6 +171,8 @@ def get_db_nft_balances(self, filter_query: 'NFTFilterQuery') -> dict[str, Any]:
         query, bindings = filter_query.prepare()
         total_usd_value = ZERO
         with self.db.conn.read_ctx() as cursor:
+            log.debug(f'QUERY: {query}')
+            log.debug(f'BINDINGS: {bindings}')
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/nft/nfts.py:175]: ARGH no. This was from somedebugging testing for the nft balances. Should go out. Let me remove.

File: rotkehlchen/tests/db/test_db_upgrades.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/tests/db/test_db_upgrades.py:1601]: assignment you mean.

But yes something is missing.

**Comment by LefterisJP** [comment]
All failed tests are due to:

1. uniswap subgraph
2. optimism etherscan having problems

**Pull request merged**